examples/step-9/step-9.cc

```diff
@@ -51,7 +51,8 @@
 // second file has a class <code>MultithreadInfo</code> (and a global object
 // <code>multithread_info</code> of that type) which can be used to query the
 // number of processors in your system, which is often useful when deciding
-// how many threads to start in parallel.
+// how many threads to start in parallel. <code>MultithreadedInfo</code> can
```

Comment:
No need to encapsulate `MultithreadInfo` into `<code>...</code>` -- doxygen will automatically get this right I believe (there are multiple places in this file -- feel free to address all of them at once).

Also watch spelling MultithreadedInfo vs MultithreadInfo